### PR TITLE
chore: add instruction for local e2e test launch

### PR DESCRIPTION
**Pull request by vrozaev**

### CODE REVIEW

File: packages/ui/README.md [hunk 1]
```diff
@@ -110,3 +110,39 @@ By default the application uses base configuration from `path_to_dist/server/con
...
+    -v $(pwd):/work \
+    "ubuntu:jammy" \
+    /bin/bash -c '
+            apt-get update
+            apt-get install curl -y
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash 
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+            nvm install v18.19.0
```
Comment (ma-efremoff) [inline packages/ui/README.md:140]: All the lines above might be removed if we use base image from our [Dockerfile](https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/Dockerfile#L1), isn't it?

File: packages/ui/README.md [hunk 2]
```diff
@@ -110,3 +110,39 @@ By default the application uses base configuration from `path_to_dist/server/con
...
+            cd tests 
+            npm ci 
+            npx playwright install
+            npx playwright install-deps
+            cd ..
+            npm run e2e:localmode:screenshots:update
+'
```
Comment (ma-efremoff) [inline packages/ui/README.md:147]:
```bash
read -p "Enter base url of your development stand: " BASE_URL ; \
docker run --rm --network host -it -w /work \
    -v $(pwd):/work \
    -e BASE_URL=${BASE_URL}
    "ubuntu:jammy" \
    /bin/bash -c '
            apt-get update
            apt-get install curl -y
            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash 
            export NVM_DIR="$HOME/.nvm"
            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
            nvm install v18.19.0
            cd tests 
            npm ci 
            npx playwright install
            npx playwright install-deps
            cd ..
            npm run e2e:localmode:screenshots:update
'
```

File: packages/ui/README.md [hunk 3]
```diff
@@ -110,3 +110,38 @@ By default the application uses base configuration from `path_to_dist/server/con
...
+
+# Terminal 1: launch local dev mode on 8080 port
+LOCAL_DEV_PORT=8080 npm run dev:localmode:e2e
+
+# Terminal 2: init local cluster
+npm run e2e:localmode:init
+
+# Terminal 2: mount repo in docker image and prepare everyting for tests run
+# Make sure that you specify correct BASE_URL for e2e tests
+# For linux use http:/localhost:8080
```
Comment (ma-efremoff) [inline packages/ui/README.md:131]: http:`//`localhost:8080

Comment (vrozaev) [reply packages/ui/README.md]: Thanks!

**Review by ma-efremoff — APPROVED** [review]

**Pull request merged**